lib/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -244,33 +283,39 @@ testRule(rule, mergeTestDescriptions(sharedAlwaysTests, {
 
   reject: [ {
     code: "a {\n\n  @mixin foo;\n  color: pink;\n}",
+    fixed: "a {\n  @mixin foo;\n  color: pink;\n}",
     message: messages.rejected,
   }, {
     code: "a {\n\n  color: pink;\n\n  @mixin foo;\n}",
+    fixed: "a {\n\n  color: pink;\n  @mixin foo;\n}",
```

Comment:
Here the spaces before `@` make sense. I think we need to distinguish these cases. If there was a space after a newline — i.e. an indentation — preserve that; but do not preserve the space when replace a space with newlines.